src/main/java/frc/robot/turret/TurretSubsystem.java

```diff
@@ -7,25 +7,34 @@
 
 import java.util.Objects;
 
+import static frc.robot.turret.TurretConstants.LOGGING_PATH;
+
 public class TurretSubsystem extends GBSubsystem {
 
-    ITurret turret;
-    TurretState state;
-    TurretInputsAutoLogged inputs;
+    private ITurret turret;
+    private TurretState state;
+    private TurretInputsAutoLogged inputs;
+    private Translation2d targetPoint;
 
     public TurretSubsystem(ITurret turret) {
```

Comment:
```suggestion
    public Turret(ITurret turret) {
```
no need of type in name